recipes/empiricaldist/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "empiricaldist" %}
+{% set version = "0.6.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/empiricaldist-{{ version }}.tar.gz
+  sha256: f7b07f5877bf5317a763e84fb5e13e06ffda66aca49e35a857d793aae68dd052
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - matplotlib-base
+    - numpy
+    - pandas
+    - seaborn
+
+test:
+  imports:
+    - empiricaldist
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/AllenDowney/empiricaldist
+  summary: Python library that represents empirical distributions.
+  license: BSD-3-Clause
```

Comment:
These files are licensed differently: 
* https://github.com/AllenDowney/empiricaldist/blob/master/empiricaldist/empiricaldist.py
* https://github.com/AllenDowney/empiricaldist/blob/master/empiricaldist/test_empiricaldist.py

Their license reference needs to be included using `AND` or (preferably) upstream needs to be updated.

Reply:
This was fixed uptream

Reply:
This was fixed uptream